site/realm/data_sources/mongodb-atlas/aiidprod/incidents/schema.json

```diff
@@ -27,6 +27,12 @@
         "description": {
             "bsonType": "string"
         },
+        "editors": {
```

Comment:
I think there is a value that can go in the schema that will reject any changes that don't include the `editors` field. Let's set that so the database integrity is defended by the schema and we don't need to figure out why the build failed later.

Reply:
In addition to "editors", I've added as required fields "_id", "incident_id", and "reports", those being the ones that wouldn't cause the schema validation to fail if required.